sdk/template/Azure.Template/src/Azure.Template.csproj

```diff
@@ -1,13 +1,27 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-  </PropertyGroup>
-  <PropertyGroup>
-    <Description>This is a template project to demonstrate how to create a package as well as use for testing our build and release pipelines</Description>
+    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
     <AssemblyTitle>Azure SDK Template</AssemblyTitle>
-    <Version>1.0.2-preview.4</Version>
+    <Version>1.0.2-preview.5</Version>
     <PackageTags>Azure Template</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableApiCompat>false</EnableApiCompat>
+    <NoWarn>$(NoWarn);AZC0001;CA1507;CA1812</NoWarn>
```

Comment:
Are these warnings a point in time or are we going to need to disable them for any library that uses the new code generation?

Reply:
AZC0001 is because `Azure.Template` is a wrong template for azure library. `CA1812` is because the simple project doesn't use all the features in shared code. CA1507 is a false positive that I don't expect to see in a real project.

Reply:
May be worth adding a comment for each of these here so folks don't end up blindly copy/pasting these.

Reply:
I'll add a comment to remove all of them in production library.